Description: Revert http://www.virtualbox.org/changeset/36310
 The libvdeplug.so symlink is only shipped in the -dev package.
Author: Felix Geyer <fgeyer@debian.org>

Index: virtualbox/include/VBox/VDEPlugSymDefs.h
===================================================================
--- virtualbox.orig/include/VBox/VDEPlugSymDefs.h
+++ virtualbox/include/VBox/VDEPlugSymDefs.h
@@ -1,5 +1,5 @@
 /** @file
- * Symbols from libvdeplug.so to be loaded at runtime for DrvVDE.cpp
+ * Symbols from libvdeplug.so.2 to be loaded at runtime for DrvVDE.cpp
  */
 
 /*
@@ -34,7 +34,7 @@
  */
 
 /** The file name of the DBus library */
-#define VBOX_LIB_VDE_PLUG_NAME "libvdeplug.so"
+#define VBOX_LIB_VDE_PLUG_NAME "libvdeplug.so.2"
 #define RT_RUNTIME_LOADER_LIB_NAME VBOX_LIB_VDE_PLUG_NAME
 
 /** The name of the loader function */
